lib/dx/evaluation.ex

```diff
@@ -23,10 +23,12 @@ defmodule Dx.Evaluation do
   end
 
   def from_options(opts) do
+    {opts, loader_opts} = Keyword.pop(opts, :loader, [])
```

Comment:
Regarding the option name, let's call it `:loader_options`, in line with naming inside Ecto, such as the `:telemetry_options` for repo functions ([docs](https://hexdocs.pm/ecto/Ecto.Repo.html#module-shared-options)).

```suggestion
    {opts, loader_opts} = Keyword.pop(opts, :loader_options, [])
```